src/Ninja/G1CP/Content/Testsuite/get.d

```diff
@@ -0,0 +1,35 @@
+/*
+ * Check if integer variable exists and return it
+ */
+func int G1CP_Testsuite_GetIntVar(var string name, var int arrIdx) {
+    var int varId; varId = G1CP_Testsuite_CheckIntVar(name, arrIdx);
+
+    return G1CP_GetIntVarI(varId, arrIdx, 0);
+};
+
+/*
+ * Check if integer constant exists and return it
+ */
+func int G1CP_Testsuite_GetIntConst(var string name, var int arrIdx) {
+    var int constId; constId = G1CP_Testsuite_CheckIntConst(name, arrIdx);
+
+    return G1CP_GetIntConstI(constId, arrIdx, 0);
+};
+
+/*
+ * Check if string variable exists and return it
+ */
+func int G1CP_Testsuite_GetStringVar(var string name, var int arrIdx) {
+    var int varId; varId = G1CP_Testsuite_CheckStringVar(name, arrIdx);
+
+    return G1CP_GetStringVarI(varId, arrIdx, 0);
+};
```

Comment:
This function should return a string and the fallback value (third argument) of `G1CP_GetStringVarI` should be a string as well. Same for `G1CP_Testsuite_GetStringConst` below.